dev/local.mk

```diff
@@ -1,12 +1,82 @@
-.PHONY: local-install
+crossplane_sentinel = $(kind_dir)/crossplane_sentinel
```

Comment:
Just as a note, we probably want to move such things into a central kind dev repo in the future.

Reply:
we need a ticket for that. I will not update the makefile for any new feature.